### PR TITLE
npy2 py313 rebuild

### PR DESCRIPTION
**Pull request by cbouss**
rebuild for numpy 2 and python 3.13

### CODE REVIEW

**Review by ELundby45 — APPROVED** [review]

File: recipe/meta.yaml [hunk 1]
```diff
@@ -35,8 +35,8 @@ outputs:
         - python
         - pip
         - freetype {{ freetype }}
-        - numpy 1.23         # [py<311]
-        - numpy {{ numpy }}  # [py>=311]
+        - numpy 2.0         # [py<313]
+        - numpy 2.1  # [py>=313]
```
Comment (danpetry) [inline recipe/meta.yaml:39]:
```suggestion
        - numpy {{ numpy }}  # [py>=313]
```

File: recipe/meta.yaml [hunk 2]
```diff
@@ -47,7 +47,7 @@ outputs:
         - cycler >=0.10
         - fonttools >=4.22.0
         - kiwisolver >=1.3.1
-        - {{ pin_compatible('numpy') }}
+        - numpy >=1.23
```
Comment (danpetry) [inline recipe/meta.yaml:50]:
```suggestion
        - numpy >=1.23,<3.0.0
```

**Review by danpetry — APPROVED** [review]
Couple of suggestions

File: recipe/meta.yaml (hunk 1 above)
Comment (cbouss) [reply recipe/meta.yaml:39]: That doesn't work. We want to build with numpy >=2 to have compatibility with both numpy 1 and 2.

File: recipe/meta.yaml (hunk 2 above)
Comment (cbouss) [reply recipe/meta.yaml]: addressed

File: recipe/meta.yaml (hunk 1 above)
Comment (danpetry) [reply recipe/meta.yaml:39]: {{ numpy }} resolves to 2.1 for py 3.13 if you have the `ANACONDA_ROCKET_ENABLE_PY313` variable set, though?

Comment (cbouss) [reply recipe/meta.yaml:39]: Ah, yes right. I read too fast and thought the suggestion was to replace both lines.

**Pull request merged**